lib/SILGen/SILGenProlog.cpp

```diff
@@ -843,7 +847,16 @@ class ArgumentInitHelper {
       }
     }
 
-    SGF.B.createDebugValueAddr(loc, value, varinfo);
+    DebugValueInst *debugInst = SGF.B.createDebugValueAddr(loc, debugValue, varinfo);
```

Comment:
Can you rename debugValue -> debugInstOperand? The reason why is that debug_value is a form of instruction so I am worried that the name debugValue for the operand is ambiguous.

Reply:
Actually, JoeG, if you move the emission of the inout mark must check here, then I think you might be able to do something smarter here.